python/paddle/fluid/tests/unittests/test_bilinear_interp_op.py

```diff
@@ -151,6 +190,8 @@ def init_test_case(self):
         self.out_h = 64
         self.out_w = 32
         self.out_size = np.array([66, 40]).astype("int32")
+        self.align_corners = False
+        self.align_mode = 0
```

Comment:
former code of blinear interpolate is base in align_corners = True? Set align_corners = True in existed tests and add align_corner = False new tests

Reply:
done